clayui.com/src/templates/docs.js

```diff
@@ -444,26 +437,15 @@ export const pageQuery = graphql`
 				}
 			}
 		}
-		allMarkdownRemark(filter: {fields: {slug: {nin: $blacklist}}}) {
+		allMarkdownRemark(filter: {fields: {slug: {in: $siblings}}}) {
 			edges {
 				node {
-					fields {
-						alwaysActive
-						layout
-						slug
-						title
-						order
-						indexVisible
-						draft
+					frontmatter {
+						tabName
 					}
+					html
 				}
 			}
 		}
-		mdTab: markdownRemark(fields: {slug: {ne: "", eq: $sibling}}) {
-			html
-		}
-		mdxTab: mdx(fields: {slug: {ne: "", eq: $sibling}}) {
-			body
-		}
```

Comment:
@kresimir-coko were you able to get some data with this query?

Some things to keep in mind.

- Queries for `.md` files are always for the `allMarkdownRemark` field.
- Queries for `.mdx` files are always for the `allMdx` field

So we must always take the two cases into consideration since we can have some tab in a `.mdx` file, we have to start from the thought that the main file or the main tab will always be the file that contains the `siblings` property and in each interaction of a new page we must search `allMdx` and `allMarkdownRemark` for these files contained in `siblings`.

Always keep in mind that this `docs.js` file will be called whenever you create a new page.